### PR TITLE
2.0 release notes

### PR DESCRIPTION
**Pull request by tdykstra**
Partially addresses #3384, will require ongoing updates.

### CODE REVIEW

File: aspnetcore/aspnetcore-2.0.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+Applications that use the `Microsoft.AspNetCore.All` metapackage automatically take advantage of the new .NET Core Runtime Store. The Runtime Store will contain all the runtime assets needed to run ASP.NET Core 2.0 applications by default. This removes the requirement to have assets from the referenced ASP.NET Core NuGet packages deployed with the application. The assets in the Runtime Store are also pre-JIT'ted to improve application startup-time.
+
+If there are features you don’t need in your application, the new package trimming features will exclude those binaries in your published application output by default.
+
+For information about the status of planned documentation, see the [metapackage GitHub issue](https://github.com/aspnet/Docs/issues/3449) and [Runtime Store GitHub issue](https://github.com/aspnet/Docs/issues/3667).
+
+## .NET Standard 2.0
+
+The ASP.NET Core 2.0 packages target .NET Standard 2.0. They can be referenced by other .NET Standard 2.0 libraries, and they can run on .NET Standard 2.0-compliant implementations of .NET, including .NET Core 2.0 and .NET Framework 4.6.1.
```
Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: They?

File: aspnetcore/aspnetcore-2.0.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+return File(data, "text/plain", "downloadName.txt", lastModified: DateTime.UtcNow.AddSeconds(-5), entityTag: entityTag);
+```
+ 
+The file returned to your visitors will be decorated with the appropriate HTTP headers for the `ETag` and `LastModified` values.
+
+If an application visitor requests content with a Range Request header, ASP.NET will recognize that and handle that header. If the requested content can be partially delivered, ASP.NET will appropriately skip and return just the requested set of bytes.  You do not need to write any special handlers into your methods to adapt or handle this feature; it is automatically handled for you.
+
+## Hosting startup and Application Insights
+
+Hosting environments can now inject extra package dependencies and execute code during application startup, without the application needing to explicitly take a dependency or call any methods. This feature can be used to enable certain environments, such as debugging in Visual Studio or hosting in Azure App Service, to "light-up" features unique to that environment without the application needing to know ahead of time.
```
Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: run on sentences. The 2nd for sure.

File: aspnetcore/aspnetcore-2.0.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+This feature is used to automatically enable Application Insights diagnostics  in ASP.NET Core 2.0 applications when debugging in Visual Studio and (after opting in) when running in Azure App Services. As a result, the project templates no longer add Application Insights packages and code by default.
+
+For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/3389).
+
+## Automatic use of anti-forgery tokens
```
Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: don't we need an issue for this? I'm documenting it for Razor pages. I didn't know MVC controllers get this too.

Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: created  #3688

File: aspnetcore/aspnetcore-2.0.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+
+ASP.NET Core has always helped HTMLEncode your content by default, but with the new version we’re taking an extra step to help prevent cross-site request forgery (XSRF) attacks: ASP.NET Core will now emit anti-forgery tokens by default and validate them on form POST actions and pages without extra configuration.
+
+## Automatic precompilation
```
Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: do we need an issue for this?

File: aspnetcore/aspnetcore-2.0.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+# What's new in ASP.NET Core 2.0 Preview
+
+> [!NOTE]
+> ASP.NET Core 2.0 is in preview, and some of the documentation for it has not yet been written. This document links to GitHub issues for articles that are planned but not yet published. For information about how to install ASP.NET 2.0 Preview 2, see [Introducing ASP.NET Core 2.0 Preview 2](https://blogs.msdn.microsoft.com/webdev/2017/06/28/introducing-asp-net-core-2-0-preview-2/)
+
+ASP.NET Core 2.0 includes the following new features:
+
+- [ASP.NET Core metapackage](#aspnet-core-metapackage)
+- [.NET Standard 2.0](#net-standard-20)
+- [Razor Pages](mvc/razor-pages/index.md)
```
Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: move this to the top

Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: Razor pages that is.

Comment (tdykstra) [inline aspnetcore/aspnetcore-2.0.md]: When used as a feature name shouldn't it be Razor Pages (capital P)?

Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: Yes

File: aspnetcore/aspnetcore-2.0.md (hunk 4 above)
Comment (tdykstra) [inline aspnetcore/aspnetcore-2.0.md]: Already have an issue, #3547

Comment (Rick-Anderson) [inline aspnetcore/aspnetcore-2.0.md]: Should we link to the issue?

Comment (tdykstra) [inline aspnetcore/aspnetcore-2.0.md]: Already added it, just haven't pushed to the remote branch yet.

**Comment by tdykstra** [comment]
@Rick-Anderson Please check the commits that came in after your review -- is this ready to go?

**Pull request merged**